project2/styles.css

```diff
@@ -0,0 +1,119 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(255, 255, 255);
+    color: rgb(0, 0, 0);
+    font-family: 'Be Vietnam', sans-serif;
+}
+
+section {
+    display: inline-block;
+    margin: auto;
+    padding-bottom: 15px;
+    width: 400px;
+
+}
+
+h2,
+ul,
+p {
+    padding-left: 5px;
+    padding-right: 5px;
+}
+
+a:link,
```

Comment:
```suggestion
```

You can just use `a` here -- the `:link` pseudoclass is only necessary when you're really minutely targeting a bunch of different states. Here, you're just using the default and the `hover`, so all should be good.